### PR TITLE
Completed error-component

### PR DESCRIPTION
**Pull request by AvinashMahanthi**
### Functionality:
Create error component

### Solution:
created the component and called the function wherever the error is returned

### Risk level:
- [x] high 
- [ ] medium
- [ ] low

### How to test:
ng serve
Click on create a new task
Submit the task without entering any details.

Screenshot (191)

### CODE REVIEW

File: worktrolly-ui/src/app/error/error.component.css [hunk 1]
```diff
@@ -0,0 +1,94 @@
+h1 {
```
Comment (shivamgoyal7) [inline worktrolly-ui/src/app/error/error.component.css:1]: Remove css from error

File: worktrolly-ui/src/app/error/error.component.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+export class ErrorComponent implements OnInit {
+
+  @Input('error') error: string;
+  constructor(private route: ActivatedRoute, private router: Router) { }
+
+  ngOnInit(): void {
+  }
+
+  closeError() {
+    this.router.navigate(['/']);
```
Comment (shivamgoyal7) [inline worktrolly-ui/src/app/error/error.component.ts:19]: Don't navigate to dashboard

Comment (shivamgoyal7) [reply worktrolly-ui/src/app/error/error.component.ts:19]: Use enable and disable here

File: worktrolly-ui/src/app/error/error.component.html [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<div class="alert alert-danger" role="alert">
+    {{error}}
+    <button type="button" class="close pull-right" aria-label="Close" (click)="closeError()">
+        <span aria-hidden="true">&times;</span>
```
Comment (shivamgoyal7) [inline worktrolly-ui/src/app/error/error.component.html:4]: For close button use material icon

File: worktrolly-ui/src/app/body/dashboard/sprint-details/sprint-details.component.ts [hunk 4]
```diff
@@ -41,6 +42,7 @@ export class SprintDetailsComponent implements OnInit {
       console.log(this.sprintStatus);
       console.log("Successfully updated Status");
     } catch (error) {
+      this.showError = true;
       console.error("Error", error);
```
Comment (shivamgoyal7) [inline worktrolly-ui/src/app/body/dashboard/sprint-details/sprint-details.component.ts:46]: Add error variable to send data

**Review by shivamgoyal7 — CHANGES_REQUESTED** [review]
Do these changes asap

**Review by viv9k — COMMENTED** [review]
Please use bootstrap model for error. Also please discuss with the stack holder for the internal functionality of errors, Since we need to implement the error codes as well.
This is not the right way to work with errors.

**Comment by viv9k** [comment]
Code design is not good and code of conduct not followed. #closed

**Pull request closed without merging**